### ISSUE TITLE
Kitchen fridge is leaking slightly

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1382]
Found another puddle in the kitchen, the tupperware box under the downpipes was not full so we pulled the fridge out and confirmed there's a pipe coming out of the back of it that had water around it, it also had some soft white stuff which came off so we binned in. Not sure of next steps, have left the fridge out so that we can get a better idea for when it happens. Probably when it automatically defrosts.

**Comment by johnmckerrell** [comment]
I wonder if the expertise of @chris-does could help here.

**Comment by johnmckerrell** [comment]
More water spillage in this area is covered by #946

**Comment by amcewen** [comment]
Bob explained that it's likely that the drainage hole in the freezer for the auto defrost is blocked.  

What's supposed to happen is that it periodically melts the ice on the sides, which should drain through a thin pipe into a tub on top of the motor.  The heat from the motor then causes the water to evaporate and everything is fine.

If/when the drainage pipe gets blocked, there's nowhere for the water to drain to, so it ends up finding its way through the seal and onto the floor.

The fix is to find a thin bit of stiff wire and poke it through the drainage hole to clean it out.

**Comment by chris-does** [comment]
> I wonder if the expertise of @chris-does could help here.

@johnmckerrell My apologies for the late reply, as I initially missed my @mention notification.

I think my luck based success with the Events Room Fridge #1234 remains substantially short of expertise, though I am happy to try and take the above information from "Bob" via @amcewen on board and investigate further.

My initial hope would be to attempt this on Thursday 19th March 2020.

**Comment by JackiePease** [comment]
There was a lot of water on the floor this morning. 
I emptied the freezer and switched it off, then unscrewed the inside back panel. Underneath was another panel with a fan attached to the front. I removed the fan (it has a circlip on the front, but this doesn't stop the fan from being taken out), and unscrewed this panel too.
Underneath was the drainage hole described by Bob, covered with a lot of ice. I poured warm water on the ice until it melted, used a length of 2.85mm filament to check that the drainage hole was now clear, then reattached the fan and panels.
The fridge is now back on - I'll check over the next couple of days to see if water is still leaking out.
